lib/checkPullRequestLabels.js

```diff
@@ -1,10 +1,30 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to handle checks when a PR gets labeled.
+ */
 const { DATASTORE_LABEL } = require('./utils');
 const {
   releaseCoordinators: RELEASE_COORDINATORS,
   SERVER_JOBS_ADMIN,
 } = require('../userWhitelist.json');
 const PR_LABELS = ['dependencies',  'stale'];
 const DEFAULT_CHANGELOG_LABEL = 'PR CHANGELOG: Miscellaneous -- @ankita240796';
+const LABELS_EXCLUDED_FROM_CODEOWNER_ASSIGNMENT = [
+  'PR CHANGELOG: Angular Migration'
+]
 
 var matchChangelogLabelWithRegex = function(changelogLabel) {
```

Comment:
Can you please add docstring for this function as well (and any others)?

Reply:
Done.